linodecli/baked/response.py

```diff
@@ -133,7 +143,7 @@ def get_string(self, model):
         return value
 
 
-def _parse_response_model(schema, prefix=None):
+def _parse_response_model(schema, prefix=None, nested_list_depth=0):
     """
     Recursively parses all properties of this schema to create a flattened set of
     OpenAPIResponseAttr objects that allow the CLI to display this response in a
```

Comment:
Can we add the parameter descriptions into the docstring below?

Reply:
Good idea! I'll add those before merging